packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/components/TransactionsCompletedRowTransferAccountDetails/TransactionsCompletedRowTransferAccountDetails.tsx

```diff
@@ -24,25 +27,28 @@ const TransactionsCompletedRowTransferAccountDetails: React.FC<TProps> = ({ acco
     if (transferAccount) {
         const derivAccountType = transferAccount === wallet ? 'wallet' : 'standard';
         const accountType = transferAccount?.platform !== 'deriv' ? transferAccount.platform : derivAccountType;
+        const mt5LandingCompanyName = transferAccount === mt5Account ? mt5Account.landing_company_name : undefined;
         const displayAccountName = getAccountName({
             accountCategory: transferAccount === wallet ? 'wallet' : 'trading',
             //@ts-expect-error this needs backend typing
             accountType,
             displayCurrencyCode: transferAccount.currency_config?.display_code ?? 'USD',
-            landingCompanyName: transferAccount.landing_company_name as TWalletLandingCompanyName,
+            landingCompanyName: activeWallet?.landing_company_name as TWalletLandingCompanyName,
```

Comment:
Doing the same as in https://github.com/binary-com/deriv-app/blob/master/packages/wallets/src/features/cashier/modules/Transfer/hooks/useExtendedTransferAccountProperties.ts#L26, i.e. using active wallet's landing company to get the name for other transfer-participating account